pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -185,8 +184,10 @@ def campaign_date(format)
     id = 'campaign_date_full_year'
   end
 
-  if %w(latam europe africa).include? type
-    id = "#{type}_#{id}"
+  # For hoc2022, we just want campaign dates for the US
+  # and non-US.
+  if @country != "us"
```

Comment:
Sorry, one other question -- does this mean countries that aren't latam, europe, or africa (eg, canada) would use the other dates? Do we want that?

Reply:
Yes, the change this year is that all non-US countries get the non-US dates, and only the US gets the US dates.  @jakebrbell can confirm.

Reply:
@breville Confirmed!